gateway/src/apicast/policy/rate_limit/apicast-policy.json

```diff
@@ -20,7 +20,7 @@
                 "description": "Limiting request concurrency (or concurrent connections)"
               },
               "key": {
-                "description": "Key of limiter",
+                "description": "The user key corresponding to the limiter object. This must be unique in the scope.",
```

Comment:
This is just a detail, but I think we should change the `user key` part.  The reason is that one of the credentials that 3scale backend accepts is also called `user key`, and it's unrelated to this. In order to avoid confusion, I'd just call it `key`.